### PR TITLE
Upgrade aws-go-sdk to 1.19.41

### PR DESCRIPTION
**Pull request by gtaylor**
We are unable to pull some of the new ELB metrics (like SpilloverCount) due to being pretty far behind in AWS SDK versions. This metric in particular has been helpful in debugging some recent issues, so we'd love to get it flowing through telegraf.

I've ran a build, the tests, and smoke tested this a bit with the plugin I'm working on. _Seems_ to be OK.

As an added bonus: this also sheds the go-ini/ini dependency, since aws-go-sdk no longer needs it (and was the sole user).

### Required for all PRs:

- [X] Signed [CLA](https://influxdata.com/community/cla/).
- [X] (Not applicable) Associated README.md updated.
- [X] (Not applicable) Has appropriate unit tests.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -56,6 +56,7 @@
 - [#5901](https://github.com/influxdata/telegraf/pull/5901): Set user agent in stackdriver output.
 - [#5885](https://github.com/influxdata/telegraf/pull/5885): Extend metrics collected from Nvidia GPUs.
 - [#5547](https://github.com/influxdata/telegraf/pull/5547): Add file rotation support to the file output.
+- [#5945](https://github.com/influxdata/telegraf/pull/5945): Upgrade from aws-sdk-go 1.15.54 to 1.19.41.
```
Comment (danielnelson) [inline CHANGELOG.md]: Can you remove this, we would only add it to the changelog if it addressed a particular bug or feature.  Also it's just a pain to avoid conflicts in this file so we usually update it post merge.

File: Gopkg.toml [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 [[constraint]]
   name = "github.com/aws/aws-sdk-go"
-  version = "1.15.54"
+  version = "1.19.41"
```
Comment (danielnelson) [inline Gopkg.toml]: Undo this change, since we don't really require a newer version than this to build.  When you open a pull request that requires the newer version you can then set it to, ideally, the first version that would support the feature you need.

The idea is to allow builders flexibility in the version they use in case they decide not to use our lock values, which is nice for system integrators.

Comment (gtaylor) [reply Gopkg.toml]: Aside from the new input we're working on, the more immediate need for the new version of the aws SDK is to pull some new ELB metrics for the existing CloudWatch input. This does not require a change to the CloudWatch telegraf input, does require a newer version of the AWS SDK.

File: CHANGELOG.md (hunk 1 above)
Comment (gtaylor) [reply CHANGELOG.md]: Dropped this.

File: Gopkg.toml (hunk 2 above)
Comment (gtaylor) [reply Gopkg.toml]: Updated the PR description to focus more on this particular issue.

Comment (danielnelson) [reply Gopkg.toml]: Can you tell me a bit more about this?  What is you want to do and what feature of the current sdk is missing?

Comment (danielnelson) [reply Gopkg.toml]: Which version is required for the SpilloverCount metric to work?
